velox/exec/tests/AggregationTest.cpp

```diff
@@ -1591,6 +1592,38 @@ TEST_F(AggregationTest, groupingSetsOutput) {
   assertEqualResults(orderResult.second, reversedOrderResult.second);
 }
 
+TEST_F(AggregationTest, groupingSetsSameKey) {
+  vector_size_t size = 5;
+  auto data = makeRowVector(
+      {"k1", "a", "b"},
+      {
+          makeFlatVector<int64_t>(size, [](auto row) { return row; }),
+          makeFlatVector<int64_t>(size, [](auto row) { return row; }),
+          makeFlatVector<StringView>(
+              size,
+              [](auto row) {
+                auto str = std::string(row % 12, 'x');
+                return StringView(str);
```

Comment:
It is not safe to return StringView over temporary memory that goes away while StringView is still alive.

Reply:
This was copied from functions above. I'll change those in another PR.

Reply:
https://github.com/facebookincubator/velox/pull/6905